Src/Sarif.PatternMatcher/AnalyzeCommand.cs

```diff
@@ -22,11 +24,31 @@ public AnalyzeCommand(IFileSystem fileSystem = null)
         {
         }
 
+        private Tool tool;
+
+        protected override Tool Tool
```

Comment:
>Tool [](http://example.com/codeflow?start=32&length=4)

@marmegh, here's the answer to your question, analyze commands now declare an explicit tool definition that is used to initialize its output, even in an API utilization context.

Reply:
Awesome! Hadn't made it this far yet. Still super excited about this change.